solidity/deploy/44_deploy_mint_burn_guard.ts

```diff
@@ -0,0 +1,38 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre
+  const { deploy, log } = deployments
+
+  const { deployer } = await getNamedAccounts()
+
+  // Allow overriding the MintBurnGuard owner and controller via env vars.
+  // Defaults:
+  // - owner: deployer
+  // - controller: unset (0x0) and configured later via governance.
+  const owner =
+    process.env.MINT_BURN_GUARD_OWNER &&
+    process.env.MINT_BURN_GUARD_OWNER.length > 0
+      ? process.env.MINT_BURN_GUARD_OWNER
+      : deployer
+
```

Comment:
Having a validation if the "owner" is a valid address would help stop the process before moving to deployment